docs/unattend.xml

```diff
@@ -16,14 +16,14 @@
          <CEIPEnabled>0</CEIPEnabled>
       </component>
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
-         <ComputerName>windows-host</ComputerName>
+         <ComputerName>winhost</ComputerName>
       </component>
    </settings>
    <settings pass="oobeSystem">
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
          <AutoLogon>
```

Comment:
Is `AutoLogon` really needed? Please see https://github.com/openshift/windows-machine-config-operator/issues/579#issuecomment-896962856

Reply:
fixed in https://github.com/openshift/windows-machine-config-operator/pull/515/commits/90d680d529aa57c2585b9a641d1d80e5d0324399